v1/protocols/ipfs.ts

```diff
@@ -166,7 +166,8 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
       // TODO: Pass in gateway parameters in options (DP domain name?)
       gateway: `https://gateway.ipfs.io/ipns/${publishKey}/`,
       cid,
-      pubKey
+      pubKey,
+      dnslink: `/ipns/${publishKey}`
```

Comment:
```suggestion
      dnslink: `/ipns/${publishKey}/`
```

IIRC there needs to be a trailing `/` in paths.